src/inet/tests/TestInetErrorStr.cpp

```diff
@@ -119,7 +118,7 @@ int TestInetErrorStr(void)
     // clang-format on
 
     // Run test suit againt one context.
-    nlTestRunner(&theSuite, &sContext);
+    nlTestRunner(&theSuite, const_cast<void *>(reinterpret_cast<const void *>(&sContext)));
```

Comment:
Oh.  This is a good reason to not make sContext const....

Reply:
Turns out that argument isn't actually used in any of the Test…ErrorStr instances.